INA181 Symbol

markus-k opened the pull request:
Add symbol for INA181 current sense amplifier. The symbol was copied from the INA282 and modified accordingly.

Datasheet for the INA181: http://www.ti.com/lit/ds/symlink/ina181.pdf

Image of the symbol:
![image](https://user-images.githubusercontent.com/1889403/43047399-132912f8-8dd7-11e8-9347-4c7f692ec06e.png)

CLAassistant commented:
[![CLA assistant check](https://cla-assistant.io/pull/badge/not_signed)](https://cla-assistant.io/KiCad/kicad-symbols?pullRequest=778) <br/>Thank you for your submission, we really appreciate it. Like many open source projects, we ask that you sign our [Contributor License Agreement](https://cla-assistant.io/KiCad/kicad-symbols?pullRequest=778) before we can accept your contribution.<br/><sub>You have signed the CLA already but the status is still pending? Let us [recheck](https://cla-assistant.io/check/KiCad/kicad-symbols?pullRequest=778) it.</sub>

evanshultz commented:
Thanks for contributing to KiCad! It's because of a willing community like you that the library can grow and improve.

You will find a list of rules and guidelines for the KiCad library at http://kicad-pcb.org/libraries/klc/. Please see that webpage for more information about the review items below.

There's only one thing to change before I can merge: Pin numbers 3 and 4 are flipped.

markus-k commented:
Whoopsie, that could've ruined someones day, thanks for pointing that out. Fixed in latest commit.

By the way, the Amplifier_Current library seems to be CR-LF, but KiCad generates LF line endings. I didn't want to change the whole file so I converted the file for the commit to match what's there. Maybe the file can be converted in a separate commit.

evanshultz commented:
Thank you!

Yes, they should all be LF (Unix style). Thanks for pointing that out. There used to be a check and auto-convert during commit on that...
